app/views/api/v2/common_parameters/base.json.rabl

```diff
@@ -1,3 +1,10 @@
 object @common_parameter
 
-attributes :id, :name, :value
+attributes :id, :name
+
+if params["show_hidden"] && User.current.allowed_to?(:edit_params)
```

Comment:
this is quite repetitive, perhaps extract to a concern that is used by both params and lks?